packages/react-charting/src/components/TreeChart/TreeChart.base.tsx

```diff
@@ -88,20 +130,34 @@ class StandardTree {
         </tspan>
       );
 
-    this._nodeElements.push(
-      <text
-        textAnchor="middle"
-        className={metricName !== undefined ? this.styleClassNames.rectSubText : this.styleClassNames.rectText}
-        dy={metricName !== undefined ? yCoordinate + rectangleHeight / 2.5 : yCoordinate + rectangleHeight / 2}
-        x={xCoordinate + rectangleWidth / 2}
-        key={`${nodeId}${this.styleClassNames.rectText}`}
-      >
-        {name}
-        {subValue}
-      </text>,
-    );
+    if (subname === undefined && metricName === undefined) {
+      this._nodeElements.push(
+        <text
+          textAnchor="middle"
+          className={this.styleClassNames.rectmetricText}
+          dy={yCoordinate + rectangleHeight / 1.6}
+          x={xCoordinate + rectangleWidth / 2}
+          key={`${nodeId}${this.styleClassNames.rectText}`}
+        >
+          {name}
+        </text>,
+      );
+    } else {
+      this._nodeElements.push(
+        <text
+          textAnchor="middle"
+          className={metricName !== undefined ? this.styleClassNames.rectSubText : this.styleClassNames.rectText}
+          id={`${nodeId}`}
```

Comment:
is the id value unique

Reply:
Yes it is unique for each node, but since we are not using selector anymore, this isn't needed

Fixed